tests/api/test_wsgi.py

```diff
@@ -0,0 +1,11 @@
+from unittest import TestCase
+from unittest.mock import patch
+
+
+class TestApi(TestCase):
+    @patch("nestor_api.api.flask_app.create_app", autospec=True)
+    def test_api_instance(self, create_app_mock):
+        """Should instance API"""
```

Comment:
You normally don't have to write a docstring.
```suggestion
```
```suggestion
        """Should instantiate API"""
```